packages/kbn-mock-idp-plugin/server/plugin.ts

```diff
@@ -40,6 +40,9 @@ export const plugin: PluginInitializer<void, void> = async (): Promise<Plugin> =
 
         const userRoles: Array<[string, string]> = [
           ['system_indices_superuser', 'system_indices_superuser'],
+          ['viewer', 'viewer'],
+          ['developer', 'developer'],
+          ['editor', 'editor'],
           ['t1_analyst', 't1_analyst'],
           ['t2_analyst', 't2_analyst'],
           ['t3_analyst', 't3_analyst'],
```

Comment:
We probably should load this roles from yml files, but I'm not sure 100%

Reply:
I added it manually so I can test my changes. I would expect mock-idp plugin to fetch the project type and display only the valid roles for running project on UI, @elastic/kibana-security wdyt?

Reply:
> I added it manually so I can test my changes. I would expect mock-idp plugin to fetch the project type and display only the valid roles for running project on UI, @elastic/kibana-security wdyt?

Yeah, we should do that, and will look into that once https://github.com/elastic/kibana/pull/172257 lands.